JavaScript/JavaScriptSDK/Logging.ts

```diff
@@ -86,6 +86,7 @@
         UrlTooLong,
         SessionStorageBufferFull,
         CannotAccessCookie,
+        IdTooLong
```

Comment:
nit: it is a good practice to have comma in the end of the list.

Reply:
Updated.